src/bundle/ui-dev/src/modules/common/thumbnail/thumbnail.tsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-
 import Icon from '../icon/icon';
 
-const Thumbnail = ({ thumbnailData, iconExtraClasses, contentTypeIconPath }) => {
-    const renderContentTypeIcon = () => {
+interface ThumbnailProps {
+    thumbnailData: {
```

Comment:
Not sure if it's the case here, but as you said on daily there's a possibility to create own types, maybe we should consider using it? I guess thumbnailData isn't the only place where we get mimeType and resource